rfcs/RFC-0003-hopr-packet-protocol/0003-hopr-packet-protocol.md

```
@@ -1,78 +1,577 @@
+   - Set `HdrExt[0]` to `HeaderPrefix_0`
+   - Copy all bytes of `Pseudonym` to `HdrExt` at offset 1
+   - Fill `HdrExt` from offset `1 + |Pseudonym|` up to `(5 - N) * RoutingInfoLen` with uniformly randomly generated bytes.
+   - Perform an exclusive-OR (XOR) of bytes generated by the PRG with HdrExt, starting from offset 0 up to `1 + |Pseudonym| + (5 - N) * RoutingInfoLen`
+   - If N > 0, generate *filler bytes* given the list of Shared secrets as follows:
+       - Allocate a zeroized buffer Filler of `(N-1)* RoutingInfoLen`
+       - For each i from 1 to N-1:
+           - Initialize new PRG instance with `SharedSecret_i`
+           - Seek the PRG to position `1 + |Pseudonym| + (4 - i) * RoutingInfoLen`
+           - XOR RoutingInfoLen bytes of the PRG to Filler from offset 0 up to `i * RoutingInfoLen`
+           - Destroy the PRG instance
+    - Copy the Filler bytes to `HdrExt` at offset `1 + |Pseudonym| + (5 - N) * RoutingInfoLen`
```

markdownlint-cli2 (0.17.2)

Inconsistent indentation for list items at the same level
Expected: 3; Actual: 4

(MD005, list-indent)